src/CONST.js

```diff
@@ -419,6 +425,7 @@ const CONST = {
         US_PHONE: /^\+1\d{10}$/,
         DIGITS_AND_PLUS: /^\+?[0-9]*$/,
         PHONE_E164_PLUS: /^\+?[1-9]\d{1,14}$/,
+        PHONE_WITH_SPECIAL_CHARS: /^[+]*[(]{0,1}[0-9]{1,3}[)]{0,1}[-\s\\./0-9]*$/,
```

Comment:
Didn't see this originally, you can ignore my suggestion to use `Str.isValidPhone`

However, this doesn't limit the number of digits after the area code. I'm not sure if it's best to always rely on using `MAX_LENGTH` in combination with `PHONE_WITH_SPECIAL_CHARS`. Does that make sense?

Reply:
What is max length of the digits after the area code?